images/runtime/node/10.1/Dockerfile.base

```diff
@@ -83,4 +83,4 @@ ARG CACHEBUST=0
 
 # PM2 is supported as an option when running the app,
 # so we need to make sure it is available in our images.
-RUN npm install -g pm2
+RUN npm install -g pm2@3.5.1
```

Comment:
how about we move this to a script that is run by every image, so we don't have to change it everywhere when we update?